org.rdkit.knime.nodes/src/org/rdkit/knime/nodes/addconformers/RDKitAddConformersNodeModel.java

```diff
@@ -322,6 +346,63 @@ protected void loadValidatedSettingsFrom(final NodeSettingsRO settings)
       catch (final InvalidSettingsException excOrig) {
          m_modelUseBasicKnowledgeOption.setBooleanValue(false);
       }
+
```

Comment:
All the code below is not necessary, I think. When loading settings for a value that is quite new (you register it with option "true" as second parameter), the normal loading mechanism will catch an InvalidSettingsException already and would keep the setting unchanged. The unchanged setting is exactly the default that we define in the Dialog class in the createSettingXXX() methods. Therefore, we should not have separate defaults defined here, unless there is a good reason (if they should be different for older nodes from current nodes).
Are you ok, if we remove all the new lines you added here? 
What about the two settings above for m_modelUseExpTorsionAnglesOption and m_modelUseBasicKnowledgeOption? I saw that we set the values to "false" for older nodes that do not know the setting, but the default we have defined for new nodes that we create is "true". Is this desired?
Keep in mind: The more we mix stuff here, the more chaotic it gets for understanding how these settings are working. Easiest would really be that the default settings are always coming directly from RDKit binaries (what you have partially added), for new nodes, but also for old nodes that do not define some settings yet.

Reply:
The idea with the way we did things with `m_modelUseExpTorsionAnglesOption` and the other one is so that existing workflows would continue to produce the same results after we added the new settings.
I think changing the behavior to what you suggest would change the results produced by old workflows which are using new versions of the nodes since the values added for the new parameters would be the defaults instead of the old values.
I realize that what we're doing now makes the code a bit more complicated, but I think it makes sense to try and preserve the behavior of old workflows whenever possible.
Do you agree with that?

Reply:
Yes, I agree with that. It's reasonable. It is kind of an "inside" knowledge of course, what the "old" values should be in that case. So, we will keep all the values that you specified as fallback.